docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -383,6 +383,8 @@ escaped-char =/ BS normal-char
 
 ; Any unicode character, except newline, double quote, and backslash
 normal-char = %x00-09 / %x0B-21 / %x23-5B / %x5D-D7FF / %xE000-10FFFF
+
+created = "ct@" timestamp
```

Comment:
ABNF is case insensitive, do you want to allow uppercase as well? if not:

```suggestion
; Lowercase ct @ timestamp
created = %d99.116 "@" timestamp
```

Reply:
Thanks for clarifying about case sensitivity

What does `%d99.116` mean?

Reply:
It's ASCII code for "ct" :)